core/frontend/src/render/RenderTarget.ts

```diff
@@ -125,6 +125,7 @@ export abstract class RenderTarget implements IDisposable, RenderMemory.Consumer
   public abstract changeRenderPlan(plan: RenderPlan): void;
   public abstract drawFrame(sceneMilSecElapsed?: number): void;
   public overrideFeatureSymbology(_ovr: FeatureSymbology.Overrides): void { }
+  public getCurrentFeatureSymbologyOverrides(): FeatureSymbology.Overrides | undefined { return undefined; }
```

Comment:
You added this, and implemented it in subclasses, but I don't see you calling it anywhere.

Reply:
Yes, that was left over from an earlier attempt, now removed.